src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0700097 MONDO:0009745 {source="OMIA:001482"} ! cross-species analog neuronal ceroid lipofuscinosis 5
+
+[Term]
+id: MONDO:1010183
+name: neuronal ceroid lipofuscinosis, 1, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 1 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001504 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001504"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001504"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009744 {source="OMIA:001504"} ! cross-species analog neuronal ceroid lipofuscinosis 1
+
+[Term]
+id: MONDO:1010184
+name: neuronal ceroid lipofuscinosis, 10, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 10 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001505 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001505"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001505"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0012414 {source="OMIA:001505"} ! cross-species analog neuronal ceroid lipofuscinosis 10
+
+[Term]
+id: MONDO:1010185
+name: neuronal ceroid lipofuscinosis, 8, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 8 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001506 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001506"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001506"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010830 {source="OMIA:001506"} ! cross-species analog neuronal ceroid lipofuscinosis 8
+
+[Term]
+id: MONDO:1010186
+name: Niemann-Pick disease, type A, non-human animal
+def: "Niemann-Pick disease, type A that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001795 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001795"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001795"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009756 {source="OMIA:001795"} ! cross-species analog Niemann-Pick disease type A
+
+[Term]
+id: MONDO:1010187
+name: neuronal ceroid lipofuscinosis, 7, non-human animal
+def: "Neuronal ceroid lipofuscinosis, 7 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001962 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001962"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:001962"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0012588 {source="OMIA:001962"} ! cross-species analog neuronal ceroid lipofuscinosis 7
+
+[Term]
+id: MONDO:1010188
+name: Niemann-Pick disease, type C2, non-human animal
+def: "Niemann-Pick disease, type C2 that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.